Changelog.md

```diff
@@ -1,6 +1,7 @@
 ### 0.7.5 (unreleased)
 
 Compiler Features:
+ * Command Line Interface: New option ``--experimental-via-ir`` allows examining generated code via the Yul intermediate representation.
```

Comment:
"examining generated code" sounds like it would do the same as ``--ir``.

Reply:
Fixed.